src/airport.js

```diff
@@ -1 +1,49 @@
+class Airport {
+    hangar;
+    #maxCapacity;
+    weather;
 
+    constructor(plane = [], capacity = 3, weather = 'Clear') {
+        this.hangar = plane;
+        this.#maxCapacity = capacity;
+        this.weather = weather;
+    }
+
+    getWeather() {
+        return this.weather;
+    }
+
+    changeWeather(weather) {
+        weather === 'Stormy' ? this.weather = 'Stormy' : weather === 'Clear' ? this.weather = 'Clear' : console.log(`Please enter valid weather conditions. Valid conditions are 'Clear' or 'Stormy'.`);
+    }
+
+    getPlaneNum() {
```

Comment:
getPlaneNum() function unnecessary, could be removed. I added this originally as hangar was set to a private property, however I changed this to public due to problems with making other functions work.